safe-persistence-async/src/main/scala/com/evolutiongaming/safeakka/persistence/async/AsyncPersistentBehavior.scala

```diff
@@ -90,6 +89,7 @@ private[async] class AsyncPersistentBehavior[S, C, E](
       def onHandlers(hs: Nel[SignalAndHandler], promise: Promise[Unit]) = {
         type R = CmdResult.NonStop[S, E]
 
+        @nowarn("cat=deprecation")
```

Comment:
why cannot we respect following deprecation?
```
  @deprecated("use `Change` instead", "3.0.0")
  final case class Records[+S, +E](
```

Reply:
`CmdResult.Records` case has to be handled in the implementation or the behaviour will be different.